tests/test_wallets.py

```diff
@@ -0,0 +1,21 @@
+import uuid
+
+from conduct.wallets import LNbitsWallet, LndHubWallet, LNPayWallet, LntxbotWallet
+
+
+def short_uuid() -> str:
+    return f"conduct_{str(uuid.uuid4())[:8]}"
+
+
+class TestTransactions:
```

Comment:
No idea how we can test these in a proper way :) I added this chain just for fun...
We could test against "expected" responses... but then we don't know if the services are really working.
And this is only for external REST services... how can we test the gRPC or local LND?

Reply:
> how can we test the gRPC or local LND?

That's actually the much saner thing to integration test since it can be totally self-contained. You "just" spin up your private bitcoin regtest network and two or more LN nodes, give them tBTC, create channels and then run your tests. That kinda sucks to set up for the first time but works (I did it on a private CI environment with C-Lightning once, I hope GitHub doesn't block bitcoind to avoid mining attacks :smile: )

Reply:
I have broken down the create methods, as a mere exercise, so at least we can test that the raw responses are what we expect.